astronomer/providers/databricks/operators/databricks.py

```diff
@@ -12,7 +12,120 @@
 from astronomer.providers.utils.typing_compat import Context
 
 
-class DatabricksSubmitRunOperatorAsync(DatabricksSubmitRunOperator):  # noqa: D101
+class DatabricksSubmitRunOperatorAsync(DatabricksSubmitRunOperator):
+    """
+    Submits a Spark job run to Databricks using the
+    `api/2.1/jobs/runs/submit
+    <https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsRunsSubmit>`_
+    API endpoint. Using DatabricksHook It makes two non-async API calls to
+    submit the run, and retrieve the run page URL. By getting the job id from the response polls for the status
```

Comment:
This sentence doesnt make sense without a comma.

"By getting the job id from the response polls for the status execute the Databricks trigger"